media/src/App.svelte

```diff
@@ -69,6 +71,7 @@
 
     $: currentCamera = orthoCamera ? camera2 : camera;
     $: currentControls = orthoCamera ? controls2 : controls;
+    $: pointCount = objectResponses.children.length;
```

Comment:
This doesn't really make sense - why not just use `objectResponses.children.length` directly? Also, it will throw an exception if `objectResponses` is null.

Reply:
I think we can update to this to be more error-proof:
```
$: pointCount = objectResponses ? objectResponses.children.length : 0;
```

Reply:
`objectResponse` is never null, it is initialized as a constified `THREE.Group` object. 

Regardless this was just some experimentation to see if a svelte reactivity bracket would provide the dynamic update I wanted. It didn't so I'll probably get rid of it.